paddlenlp/trainer/trainer.py

```diff
@@ -1139,8 +1139,15 @@ def _print_timer(self):
         else:
             timer_info = ""
 
+        if paddle.device.is_compiled_with_cuda():
```

Comment:
```suggestion
        if self.timers is not None and paddle.device.is_compiled_with_cuda():
```
避免不打开时候，每次都调用这个API，影响性能。

Reply:
done